.gitignore

```diff
@@ -48,6 +48,10 @@ tmp
 
 # Distribution folders
 dist/
+dist-node/
+dist-browser/
```

Comment:
Do these need to be preceded with `*/**`?

Reply:
```
If the pattern ends with a slash, it is removed for the purpose of the following description, but it would only find a match with a directory. In other words, foo/ will match a directory foo and paths underneath it, but will not match a regular file or a symbolic link foo (this is consistent with the way how pathspec works in general in Git).
```
It finds child folders when the pathname ends with /. However, doesn't do it for files. That's why i set it like this.